internal/lookout/ui/src/services/MockJobService.ts

```diff
@@ -0,0 +1,122 @@
+import { v4 as uuidv4 } from "uuid"
```

Comment:
Oh man!  This is great!  Thanks for adding this.  Its been on my list of things to add so we can troubleshoot other API errors.

Reply:
So is this mock service used anywhere else?  I'd expect to see a unit test or something using it?